### PR TITLE
[WIP] - feat(CI): add a Make workflow for testing Go with 'vet' & 'fmt'

### PR DESCRIPTION
**Pull request by jbampton**
- Add Apache license to all YAML files in the `.github` folder and sub folder `workflows`

### What this PR does / why we need it:

Need to test the Golang code.
Vet and fmt are standard Go tools.

https://golang.org/cmd/vet/
https://blog.golang.org/gofmt

### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: .github/workflows/chaos.yml [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 name: Chaos Test
 
 on: [pull_request]
 
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: ['1.15', '1.14']
```
Comment (spacewander) [inline .github/workflows/chaos.yml:27]: I don't like this idea. The Go file is only for test. And we are not interested in testing the Chao test in different Go versions. Don't make thing so much complex.

**Review by spacewander — CHANGES_REQUESTED** [review]

**Comment by spacewander** [comment]
APISIX is a gateway built with OpenResty (C + Lua).
There is no need to invest too much resource to keep the Go test code tidy.

**Comment by jbampton** [comment]
Hey @spacewander what about the other part of this pull request adding the licenses ?

Add Apache license to all YAML files in the `.github` folder and sub folder `workflows`.

**Comment by spacewander** [comment]
For that, you can create another PR to do it.

**Pull request closed without merging**